core/src/test/resources/Validation/Basic/xsd/A_without_targetnamespace.xsd-after-adding-namespace.xsd

```diff
@@ -5,7 +5,7 @@
     >
   <!-- targetNamespace is missing, this is added by the validator configuration -->
 
-  <xs:import namespace="http://xmlns/b" schemaLocation="B.xsd" />
+  <xs:import namespace="http://xmlns/b"/>
```

Comment:
Removed, because the production code strips the schema location but the previous test-only method didn't.